ReSharper.FSharp/src/FSharp/FSharp.Psi.Features/src/Parsing/FSharpTreeBuilderBase.fs

```diff
@@ -941,22 +946,34 @@ type FSharpTreeBuilderBase(lexer: ILexer, document: IDocument, warnDirectives: W
         x.MarkAndDone(range, ElementType.TYPE_PARAMETER_ID)
         x.Done(range, mark, ElementType.TYPE_REFERENCE_NAME)
 
-    member x.ProcessAccessorsNamesClause(trivia: SynValSigTrivia, memberRange) =
-        match trivia.WithKeyword with
-        | None -> ()
-        | Some withRange ->
+    member x.ProcessAccessorNameWithModifiers(accessorNameRange: range) =
+        x.AdvanceToAnyInTokenSetOrPos(FSharpTokenType.AccessModifiers, accessorNameRange.Start)
```

Comment:
This approach comes from the times when it wasn't possible to rely on the compiler tree ranges. Since then many ranges have been fixed and we have a good parser testing framework in the compiler in place.

Would it be possible to provide the needed range in the FCS tree instead of seeking tokens here?